### PR TITLE
cloud: remove web sql shell from dev tier

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: remove web sql shell from dev tier

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 1]
```diff
@@ -127,6 +127,12 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
+> **Note:**
+>
+> SQL shell is only available for Dedicated Tier clusters, not for Developer Tier clusters.
+>
+> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access it, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select **Edit and Run Your SQL** tab.
```
Comment (TomShawn) [inline tidb-cloud/connect-to-tidb-cluster.md:134]:
```suggestion
> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access it, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select the **Edit and Run Your SQL** tab.
```

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 2]
```diff
@@ -127,6 +127,12 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
+> **Note:**
+>
+> SQL shell is only available for Dedicated Tier clusters, not for Developer Tier clusters.
+>
+> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access it, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select the **Edit and Run Your SQL** tab.
```
Comment (ran-huang) [inline tidb-cloud/connect-to-tidb-cluster.md:134]:
```suggestion
> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access **Playground**, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select the **Edit and Run Your SQL** tab.
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 3]
```diff
@@ -127,6 +127,12 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
+> **Note:**
+>
+> SQL shell is only available for Dedicated Tier clusters, not for Developer Tier clusters.
+>
+> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access **Playground**, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select the **Edit and Run Your SQL** tab.
```
Comment (ran-huang) [inline tidb-cloud/connect-to-tidb-cluster.md:134]: How about moving this to L127, just under the heading?

Comment (TomShawn) [reply tidb-cloud/connect-to-tidb-cluster.md:134]: +1

**Pull request closed without merging**